### PR TITLE
Update Project.toml

### PR DESCRIPTION
**Pull request by rofinn**
Update Project.toml to support a new release via the registrator app.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 name = "CloudWatchLogs"
 uuid = "c4c1e6a2-6bdd-52e0-b56d-1d4734724d2d"
+authors = ["Invenia Technical Computing"]
```
Comment (nicoleepp) [inline Project.toml]: Is our standard `Invenia Technical Computing` or `Invenia Technical Computing Corporation`?

Comment (rofinn) [reply Project.toml]: I'm not sure, the latter seems rather long.

Comment (ararslan) [reply Project.toml]: I opted to avoid adding that section to Nabla's Project.toml pending further clarification from The Officials.

Comment (iamed2) [reply Project.toml]: Standard is "Invenia Technical Computing Corporation" as recorded in InveniaTemplates.jl

**Comment by rofinn** [comment]
I was going to leave it for a release just to make sure the Project.toml hasn't fallen out of date.

**Pull request merged**